tools/Requirements-Checker/checker.php

```diff
@@ -206,6 +206,13 @@
 );
 
 $tests[] = array(
+	'title' => 'Intl extension',
+	'required' => FALSE,
+	'passed' => class_exists('Transliterator', FALSE),
+	'description' => 'Class Transliterator for script transliteration in Nette\Utils\Strings::webalize() and toAscii().',
```

Comment:
Can you change the message to match the above ones? Something like _"Class Transliterator is absent, the output of Nette\Utils\Strings::webalize() and Nette\Utils\Strings::toAscii() may not be accurate."_